src/makers/darwin/dmg.js

```diff
@@ -10,9 +10,9 @@ export default async (dir, appName, targetArch, forgeConfig, packageJSON) => { /
   await ensureFile(outPath);
   const dmgConfig = Object.assign({
     overwrite: true,
+    name: appName,
```

Comment:
@malept I made it a separate commit, noticed it while making this PR 😆   Was trying to rename artifacts in the postMake hook while testing and saw the artifact was named incorrectly 😆